opm/autodiff/GeoProps.hpp

```diff
@@ -111,21 +111,8 @@ namespace Opm
             // Get grid from parser.
             EclipseGridConstPtr eclgrid = eclState->getInputGrid();
 
-            // Pore volume.
-            // New keywords MINPVF will add some PV due to OPM cpgrid process algorithm.
-            // But the default behavior is to get the comparable pore volume with ECLIPSE.
-            for (int cellIdx = 0; cellIdx < numCells; ++cellIdx) {
-                int cartesianCellIdx = AutoDiffGrid::globalCell(grid)[cellIdx];
-                pvol_[cellIdx] =
-                    props.porosity()[cellIdx]
-                    * multpv[cartesianCellIdx]
-                    * ntg[cartesianCellIdx];
-                if (eclgrid->getMinpvMode() == MinpvMode::ModeEnum::OpmFIL) {
-                    pvol_[cellIdx] *= AutoDiffGrid::cellVolume(grid, cellIdx);
-                } else {
-                    pvol_[cellIdx] *= eclgrid->getCellVolume(cartesianCellIdx);
-                }
-            }
+            // update the pore volume of all active cells in the grid
+            computePoreVolume_(grid, eclState);
```

Comment:
I think default should be to just just use the raw pv as computed by EclipseState. The modified pv should only be used when MinpvMode::ModeEnum::OpmFIL.